examples/rstudio/test.bats

```diff
@@ -0,0 +1,32 @@
+true
+# shellcheck disable=SC2034
+CH_TEST_TAG=$ch_test_tag
+
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope full
+    prerequisites_ok "rstudio"
+}
+
+@test "${ch_tag}/test rstudio-server" {
+    # Start up on port 8991
+    run ch-run "$ch_img" -- start_rstudio 8991 &
```

Comment:
Why port 8991?

Reply:
I chose it arbitrarily. As far as I can tell, no application generally use it. I had tried writing a script that looks for an unused port and uses that, but it's not really possible to do that in bash without race conditions since you can't access the socket interface directly. We could use `8080` which I believe is the canonical port for something like this but a lot of applications use it as their default port.